libraries/lib-graphics/CMakeLists.txt

```diff
@@ -5,6 +5,12 @@ A library responsible for custom rendering in Audacity
 set( SOURCES
    FrameStatistics.cpp
    FrameStatistics.h
+
+   graphics/Color.cpp
+   graphics/Color.h
+   graphics/Point.h
+   graphics/Rect.h
+   graphics/Size.h
```

Comment:
I would put Point, Rect, and Size in lib-screen-geometry which can be just about the geometry with no mention of color.  That's a leaf library not even using lib-utility.

Yes, lib-screen-geometry still has extraneous constants about the particular Audacity 3 layout, as we discussed before.   Let me figure out how to clean that up.

Reply:
The "problem" with this specific PR is that it is two years old and predates many of the libraries. I haven't re-reviewed much myself, but most, if not all, of the lib-graphics are likely not strictly needed for the purposes of the pull request. 

I am not sure that we want it for 3.5 or for Audacity 3 at all; this was never discussed @ianalarina. Please note that the PR is not even on the board.